lutris/gui/widgets/game_bar.py

```diff
@@ -206,11 +213,11 @@ def get_play_button(self, game_actions):
 
         if self.game.is_installed:
             game_buttons = self.get_game_buttons(game_actions)
-            if self.game.state == self.game.STATE_STOPPED:
+            if self.game.game_launcher.state == self.game.game_launcher.STATE_STOPPED:
```

Comment:
This is like the GameActions, and should be accessing the running games like that does. Or maybe it should actually be the same code somehow?